src/program.js

```diff
@@ -86,6 +86,7 @@ program
   .option('--minProfitability <eth>', 'Only claim transactions with a bounty higher', Config.DEFAULT_ECONOMIC_STRATEGY.minProfitability.div(Math.pow(10, 18)))
   .option('--maxGasSubsidy <eth>', 'Subsidize a percentage of gas costs on gas spikes', Config.DEFAULT_ECONOMIC_STRATEGY.maxGasSubsidy)
   .option('--ms <number>', 'Sets the scanning frequency of the TimeNode', 4000)
+  .option('--providers <list_of_providers>', '')
```

Comment:
How does one provide this array? A string separate by something?

Reply:
Oh whoops. Yeah it should be a string of the addresses separated by commas, no spaces.

Reply:
Please add a description too. Keep in mind that commas can be a part of an URL. We might want to use a different separator.

Reply:
Hm... That's true, although they're rarely used. Adding in Config compatibility too in the case where the comma separated string doesn't work.